target_chains/solana/sdk/js/pyth_solana_receiver/src/vaa.ts

```diff
@@ -47,7 +76,24 @@ export async function buildEncodedVaaCreateInstruction(
   };
 }
 
-export async function buildWriteEncodedVaaWithSplit(
+/**
+ * When we are writing to an encoded VAA account, we do it in two instructions.
```

Comment:
suggest "the byte index to split VAAs..." to eliminate the need for "we" here.

Reply:
fixed